opencraft/settings.py

```diff
@@ -466,6 +466,10 @@
 BETATEST_EMAIL_SENDER = env('BETATEST_EMAIL_SENDER', default='betatest@example.com')
 BETATEST_EMAIL_SIGNATURE = env('BETATEST_EMAIL_SIGNATURE', default='The Beta Test Team')
 
+# Determine whether to update server status whenever there's an API call requesting it
+
+UPDATE_OPENSTACK_SERVER_STATUS_ON_API_CALL = env.bool('UPDATE_OPENSTACK_SERVER_STATUS_ON_API_CALL', default=True)
```

Comment:
I'd rather avoid adding configuration options for which we don't have an express need. Having code adopting different behaviors tend to increase the maintenance burden - especially when tests only test one of the configurable options. Adding one to not have to change tests, especially, is a red flag imho - I haven't looked at the tests in question, but mocking can be a more robust approach, as it's explicit for each test.